src/pages/index.js

```diff
@@ -3,8 +3,10 @@ import FormValidator from "../components/FormValidator.js";
 import "./index.css";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import confirmDelete from "../components/ConfirmDelete.js";
```

Comment:
`confirmDelete ` should be imported in **UpperCase**, because it's a **class**


```js

import ConfirmDelete  from "../components/ConfirmDelete.js";

```